op-batcher/batcher/driver.go

```diff
@@ -324,6 +329,71 @@ func (l *BatchSubmitter) loop() {
 	}
 }
 
+// checkRecentTxsOnStart Check to see if there was a batcher tx sent recently that
+// still needs more block confirmations before being considered finalized
+func (l *BatchSubmitter) checkRecentTxsOnStart() {
+	if !l.Config.CheckRecentTxsOnStart {
```

Comment:
Do you think its better to have this outside of the function rather than inside? Feels like its easier to test that way

Reply:
Yes that would make it easier to test, at least at the unit test level that this PR implements. Will make that change

Reply:
Updated in https://github.com/ethereum-optimism/optimism/pull/10193/commits/add2881fd459ad8c34fb94ffe0b70802a6f2b1b1